src/NuGet.Core/NuGet.Packaging/PackageCreation/Authoring/ManifestReader.cs

```diff
@@ -304,12 +304,12 @@ private static List<ManifestFile> ReadFilesList(XElement xElement)
                     continue;
                 }
 
-                string target = file.GetOptionalAttributeValue("target").SafeTrim();
+                string target = file.GetOptionalAttributeValue("target").SafeTrim()?.TrimStart(new[] { '\\', '/' });
                 string exclude = file.GetOptionalAttributeValue("exclude").SafeTrim();
 
                 // Multiple sources can be specified by using semi-colon separated values. 
                 files.AddRange(from source in srcElement.Value.Trim(';').Split(';')
-                               select new ManifestFile { Source = source.SafeTrim(), Target = target.SafeTrim(), Exclude = exclude.SafeTrim() });
+                               select new ManifestFile { Source = source.SafeTrim().TrimStart(new[] { '\\', '/' }), Target = target, Exclude = exclude });
```

Comment:
This line of code is too cluttered. I'd split it into multiple lines. Not required though.